changelog/4074.breaking.rst

```diff
@@ -0,0 +1 @@
+`sunpy.cm` is now removed, please use `sunpy.visualization.colormaps.cm` instead.
```

Comment:
I thought you said we only deprecated this in 1.1. Also breaking is the wrong type.

Reply:
Oh I forgot to remove it.